apps/files/src/components/FileEntry/FileEntryPreview.vue

```diff
@@ -14,16 +14,19 @@
 			</template>
 		</template>
 
-		<!-- Decorative image, should not be aria documented -->
-		<img v-else-if="previewUrl && backgroundFailed !== true"
-			ref="previewImg"
-			alt=""
-			class="files-list__row-icon-preview"
-			:class="{'files-list__row-icon-preview--loaded': backgroundFailed === false}"
-			loading="lazy"
-			:src="previewUrl"
-			@error="onBackgroundError"
-			@load="backgroundFailed = false">
+		<!-- Decorative images, should not be aria documented -->
+		<span v-else-if="previewUrl" class="files-list__row-icon-preview-container">
+			<canvas v-if="hasBlurhash && (backgroundFailed === true || !backgroundLoaded)" ref="canvas" class="files-list__row-icon-blurhash" aria-hidden="true"/>
+			<img v-if="backgroundFailed !== true"
```

Comment:
Prevent that both are shown, or am I missing something here?

```suggestion
			<img v-else-if="backgroundFailed !== true"
```

Reply:
No, we need both at the beginning, else the preview is never loaded by the `<img>`. The canvas will then be hidden when the preview is loaded.